sycl/test/regression/memcpy-update-leafs.cpp

```diff
@@ -1,44 +0,0 @@
-// RUN: %clangxx -fsycl %s -o %t.out
-// RUN: env SYCL_PI_TRACE=1 %CPU_RUN_PLACEHOLDER %t.out 2>&1 %CPU_CHECK_PLACEHOLDER
```

Comment:
Removed the test because it expects line(piEnqueueMemBufferWrite) which is not generated for the example anymore. Attempt to rewrite the test so was not successful as the problem this test was written for cannot appear anymore.